File: ruby/src/Behavioral/Strategy/DvdNameAllCapStrategy.rb

## DvdNameAllCapStrategy - one of three concrete strategies

#require "Behavorial/Strategy/DvdNameStrategy"

class DvdNameAllCapStrategy < DvdNameStrategy

	def initialize
	end

	
	def formatDvdName( dvdName, charIn)
		dvdName.upcase
	end

end

File: csharp/src/Behavioral/Strategy/DvdNameAllCapStrategy.cs
namespace Behavioral.Strategy {


//DvdNameAllCapStrategy.java - one of three concrete strategies

public class DvdNameAllCapStrategy : DvdNameStrategy {
  override public string formatDvdName(string dvdName, char charIn) {
       return dvdName.ToUpper();
   }
}

}
